### ISSUE TITLE
Compendium main window cannot be resized

### ISSUE THREAD

**Issue by SebastianBoell** [CompendiumNG/CompendiumNG#64]
http://compendium.open.ac.uk/bugzilla/show_bug.cgi?id=925

Michelle Bachler 2010-09-14 10:08:38 BST

Reported by Paul Hughes-Barlow:

I am running Vista and Compendium Version 2 beta 1. The problem I am having is
that I am unable to resize the window which makes it impossible to drag and
drop files or folders.

Paul

Hi!

That is very strange. I am not sure I have heard this issue before.
What actually happens if you click the Window expand/contract button at the top
right hand corner?
Can you just not drag the corner of the Window?

Possible test:
In <compendium dir>/System/resources there is a file called Format.properties.
If you open that with a text editor, you will see lots of saved properties.
Two of them remember the last window size of Compendium and restore Compendium
to that window size next time.

lastScreenWidth=
lastScreenHeight=

It does not prevent resizing, it just restores stuff to the way you had it.
I thought, however, you could try changing the sizes to something small, then
start Compendium and see what happens.

Michelle

**Comment by SebastianBoell** [comment]
This seems to be platform independent. I tried to reproduce:
1) maximize the Compendium window.
2) close Compendium
3) reopen Compendium

Now double clicking the window frame no longer resizes the Compendium Window also the Icon for resizing changes once from 'maximized' to 'window' mode but further double clicking does not change it again. However, the behaviour changes after every double cklick. As a result resizing the maximized window only works every second time (as one time the window is maximizes and subsequently is not resizable).

Making sure that Compendium never starts with maximal screen dimensions should solve this issue. That is on startup Compendium should always be in windowed mode on a size smaller than the physical resolution of the screen. Of course this is a bit of a dodgy, but working solution to the problem.

**Comment by maiklos** [comment]
I am not sure if you managed to reproduce the same behavior.

i.e. on LXDE this described behavior is same. When application window is maximized then it can't be changed by double clicking frame. I suspect this is window manager specific.

on the other side when I used gnome+metacity it was possible to resize the frame by double-click.

in both cases another reference window (firefox) manifested exactly same behavior.

**Comment by SebastianBoell** [comment]
Working with Gnome and Metacity, but I did not notice the same behaviour regarding another window. Will keep my eyes open.

Mark this as (potentially) platform specific to indicate that we should do some testing in different settings.

**Comment by maiklos** [comment]
I tried to reproduce the problem with the same result as above. This is minor issue if at all. I lowered the severity

**Comment by SebastianBoell** [comment]
I agree this is a minor issue not affecting functionality. As stated above I believe if we can make sure that CNG is  starting in windowed mode with resolution smaller than the physical resolution of the screen this bug can be completely resolved.
